packages/cli/internal/pkg/cli/account_activate.go

```diff
@@ -147,6 +188,7 @@ func displayProgress(progressStream cdk.ProgressStream, displayMsg string) error
 			}
 			lastEvent = event
 		}
+		log.Info().Msg(strings.Join(lastEvent.Outputs, "\n"))
```

Comment:
I just want to verify this is an intended change and not some debugging log.

Reply:
running `account activate -v` will return a neat set of outputs including the TableName, VpcId, and Bucket. 
![Screen Shot 2022-03-08 at 2 09 08 PM](https://user-images.githubusercontent.com/52179349/157563303-c4ab5eb7-41cf-4cad-a0eb-6e021f77358f.png)